tests/pipeline/test_pipeline.py

```diff
@@ -5,23 +5,33 @@
 from snapshottest.module import SnapshotTest
 from typer.testing import CliRunner
 
+import kpops
 from kpops.cli.main import app
 from kpops.pipeline_generator.pipeline import ParsingException
 
 runner = CliRunner()
 
 RESOURCE_PATH = Path(__file__).parent / "resources"
```

Comment:
os.join or use Path for joining it already ?

Reply:
what do you mean? the `/` operator works the same as `Path(...).join(...)`